plugins/ptp_operator/metrics/metrics.go

```diff
@@ -131,6 +131,7 @@ func (p *PTPEventManager) ExtractMetrics(msg string) {
 	//  check if ptp4l or phc2sys process are dead
 	// if process is down fire an event
 	if strings.Contains(output, ptpProcessStatusIdentifier) {
+		log.Infof("output %s", output)
```

Comment:
is this temporary? too much logs.

Reply:
yes , it was debug log . Thanks will remove it